recipes/tensorflow-text/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "2.17.0" %}
+{% set tensorflow_version = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tensorflow-text
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
+  sha256: 55bff24f120024664124c8fdc0ab842ab314bd5ff2791770a80f32bb67cb26d1
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+
+requirements:
+  build:
+  - {{ compiler('c') }}
+  - {{ stdlib('c') }}
+  - {{ compiler('cxx') }}
+  - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+  - bazel >=5.2.0
+  host:
+  - python
+  - pip
+  - packaging
+  - setuptools >=70
+  - wheel
+  - absl-py
+  - tensorflow {{ tensorflow_version }}
+  - libtensorflow *=cuda*  # [cuda_compiler_version != "None"]
+  - libtensorflow *=cpu*   # [cuda_compiler_version == "None"]
```

```suggestion
  - tensorflow-base *=cuda*  # [cuda_compiler_version != "None"]
  - tensorflow-base *=cpu*   # [cuda_compiler_version == "None"]
```

sorry i think i got the name of the package wrong.

---

Done.